src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -0,0 +1,43 @@
+import path from 'ramda/src/path';
+import { buildATIPageTrackPath } from '../../atiUrl';
+import { getPublishedDatetime } from '../../../../lib/analyticsUtils';
+
+export const buildCPSATIParams = (pageData, requestContext, serviceContext) => {
+  const { platform, statsDestination } = requestContext;
+  const {
+    atiAnalyticsAppName,
+    atiAnalyticsProducerId,
+    service,
+  } = serviceContext;
+
+  const { metadata, promo } = pageData;
+
+  // ATI gets it's "Chapter 1" value from a prefix to the page identifier
+  // eg embedded_media::pidgin.embedded_media.media_asset.49529724.page
+  const page = path(['analyticsLabels', 'counterName'], metadata);
+  const isValidPage = page && typeof page === 'string' && page.includes('.');
+  const chapter1 = isValidPage && page.split('.')[1];
+
+  return {
+    appName: atiAnalyticsAppName,
+    contentId: metadata.id,
```

Comment:
Will this error if `metadata` is undefined from the destructing on line 13? If so it can be `path()` ... same for `metadata.language`